moduleframework/common.py

```diff
@@ -252,29 +203,18 @@ def get_odcs_auth():
     :envvar MTF_ODCS: yes or token
     :return:
     """
-    odcstoken = os.environ.get('MTF_ODCS')
+    odcstoken = get_odcs_envvar()
 
     # in case you dont have token enabled, try to ask for openidc via web browser
-    if odcstoken in TRUE_VALUES_DICT:
+    if is_true(odcstoken):
+        if conf.get("openidc").get("token"):
+            # use value defined in config file if defined
+            return conf["openidc"]["token"]
```

Comment:
`get` and `get` are missing.

Reply:
@phracek it is dict, so it is not important to use get, because it is already checked that is defined in `if`